client/javascript/pages/Dashboard.jsx

```diff
@@ -141,6 +168,7 @@ Dashboard.propTypes = {
 };
 
 Dashboard.defaultProps = {
+  title: 'Dashboard',
```

Comment:
Does this need to be a prop each time? You could just pass the string to the document title component